prime-dotnet-webapi/Services/EnrolleeAgreementService.cs

```diff
@@ -124,6 +130,16 @@ public async Task<Agreement> GetCurrentAgreementAsync(int enrolleeId)
                 .FirstAsync(at => at.EnrolleeId == enrolleeId);
         }
 
+        public async Task<AgreementType> GetCurrentAgreementTypeAsync(int enrolleeId)
```

Comment:
This needs to be `AgreementType?` since users without `Agreements` have no types